### PR TITLE
Implemented routing for custom todo component

### PR DESCRIPTION
**Pull request by sashkivna**

### CODE REVIEW

File: app/app.module.config.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            name: 'login',
+            url: '/login',
+            templateUrl: 'login/register-template.html',
+        };
+
+        $stateProvider.state(mainState);
+        $stateProvider.state(loginState);
+        $stateProvider.state(usersState);
+    })
+    .run(function ($rootScope, $location) {
```
Comment (r0ss0neri) [inline app/app.module.config.js]: create separate file `app.run.js`

File: app/app.module.config.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+angular.module('app.module')
```
Comment (r0ss0neri) [inline app/app.module.config.js]: rename file `app.module.config.js` -> `app.config.js`

File: app/app.module.config.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    })
+    .run(function ($rootScope, $location) {
+        $rootScope.$on('$stateChangeStart',
+            function (evt, next) {
+                if (!$rootScope.currentUser) {
+                    $location.path('/login');
+                } else {
+                    $location.path('/main');
+                }
+
```
Comment (r0ss0neri) [inline app/app.module.config.js]: remove blank line, format code

Comment (r0ss0neri) [reply app/app.module.config.js]: $rootScope.$on('$stateChangeStart', function () {
        if ($rootScope.currentUser) {
            $state.go('main');
        } else {
            $state.go('login');               
        }

File: app/users-todo/users-todo.controller.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+angular.module('app.module')
+    .controller('usersCntr', ['$rootScope', '$scope', 'appCasheService', '$state', function ($rootScope, $scope, appCasheService, $state) {
+        $rootScope.currentUser = 'vasya';
+        $scope.usersFromCashe = appCasheService.getUsersFromCashe();
```
Comment (r0ss0neri) [inline app/users-todo/users-todo.controller.js:3]: blank line after

File: app/users-todo/users-todo.controller.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+angular.module('app.module')
+    .controller('usersCntr', ['$rootScope', '$scope', 'appCasheService', '$state', function ($rootScope, $scope, appCasheService, $state) {
```
Comment (r0ss0neri) [inline app/users-todo/users-todo.controller.js]: refactor second parameter in controller to use function instead of array of arguments

**Review by r0ss0neri — CHANGES_REQUESTED** [review]

File: app/app.module.config.js (hunk 1 above)
Comment (sashkivna) [reply app/app.module.config.js]: done

File: app/app.module.config.js (hunk 2 above)
Comment (sashkivna) [reply app/app.module.config.js]: done

File: app/app.module.config.js (hunk 3 above)
Comment (sashkivna) [reply app/app.module.config.js]: done

File: app/users-todo/users-todo.controller.js (hunk 4 above)
Comment (sashkivna) [reply app/users-todo/users-todo.controller.js:3]: done

File: app/users-todo/users-todo.controller.js (hunk 5 above)
Comment (sashkivna) [reply app/users-todo/users-todo.controller.js]: done

File: package.json [hunk 6]
```diff
@@ -17,9 +17,11 @@
   },
   "homepage": "https://github.com/sashkivna/viewRendering#readme",
   "dependencies": {
+    "@uirouter/angularjs": "^1.0.20",
     "angular": "^1.7.4",
     "angularjs": "0.0.1",
-    "jquery": "^3.3.1"
+    "jquery": "^3.3.1",
+    "ngstorage": "^0.3.11"
```
Comment (r0ss0neri) [inline package.json]: what is `ngstorage`?

**Review by r0ss0neri — CHANGES_REQUESTED** [review]
remove debugger every where

File: package.json (hunk 6 above)
Comment (sashkivna) [reply package.json]: debugger removed.
deleted ngStorage from package.json

**Pull request merged**